test/correctness/compute_with.cpp

```diff
@@ -0,0 +1,745 @@
+#include "Halide.h"
+#include "../common/check_call_graphs.h"
```

Comment:
Please avoid using relative include paths with #include; it makes working with Bazel harder than it could be. Use the pattern we're now using elsewhere: 

     #include "test/common/check_call_graphs.h"